Misc/NEWS.d/next/C API/2023-02-06-16-14-30.gh-issue-101578.PW5fA9.rst

```diff
@@ -1,13 +1,9 @@
-Add new C-API functions for saving and restoring the current exception:
-``PyErr_GetRaisedException`` and ``PyErr_SetRaisedException``.
-These functions take and return a single exception rather than
-the triple of  ``PyErr_Fetch`` and ``PyErr_Restore``.
+Add :c:func:`PyErr_GetRaisedException` and :c:func:`PyErr_SetRaisedException`
+for saving and restoring the current exception.
+These functions take and return a single exception,
```

Comment:
```suggestion
These functions return and accept a single exception object,
```

As above, or whatever is a more appropriate word than "object" to describe an instance of the `PyException` type as opposed to an exception as a higher-level construct.